vendor/licenses/git_submodule/typst-grammar.dep.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: typst-grammar
+version: 8bc1b547d82853bcdd09c56918d9cff2d653fef7
+type: git_submodule
+homepage: https://github.com/michidk/typst-grammar.git
+license: mit
+licenses:
+- sources: LICENSE
+  text: |
+    The MIT License (MIT)
+    Copyright (c) 2023 Nathan Varner
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+    The above copyright notice and this permission notice shall be included in all
+    copies or substantial portions of the Software.
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+notices: []
```

Comment:
Your fork of the grammar should really have the MIT license text in the `LICENSE` or `LICENSE.md` file in the root of the repo as the upstream grammar's directory has a [`LICENSE` file](https://github.com/nvarner/typst-lsp/blob/master/addons/vscode/LICENSE.md) which states:

> typst-lsp is dual-licensed under either
> 
> -   MIT License ([../../LICENSE-MIT.txt](../../LICENSE-MIT.txt) or http://opensource.org/licenses/MIT)
> -   Apache License, Version 2.0 ([../../LICENSE-APACHE-2.0.txt](../../LICENSE-APACHE-2.0.txt) or http://www.apache.org/licenses/LICENSE-2.0)
> 
> at your option.

This will also allow the Linguist license caching to automatically keep up to date with changes without manual intervention each time.